Cargo.toml

```diff
@@ -33,7 +33,6 @@ exclude = [
   "ci/benchmarks/bench-runner",
   "ci/repl_basic_test",
   # Examples sometimes have Rust hosts in their platforms. The compiler should ignore those.
-  "crates/cli_testing_examples",
```

Comment:
`find crates/cli/tests -type f -name "Cargo.toml"` didn't turn up anything, so I think its ok to remove this?

Reply:
Yeah, they all use zig platforms now, so it's fine to remove that.